app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -143,6 +148,25 @@ textarea {
   float: none;
 }
 
+// Must reside in the top level of stylesheets
+// Note: sticky-footer is setting the height of the first selector to auto !important. To counteract that we are setting it manually.
+@include sticky-footer($footer-height, "#foo-bar", "#layout_footer", "#footer");
+#content {
```

Comment:
The idea is that `//` comments are SCSS comments, and don't show up in the compiled CSS, while `/* */` are CSS comments, and stay in the compiled CSS. There's precious bits to shave off!

I also read something once about why you should use inline comments over block comments, something to do with it being able to keep track of where the comment starts and ends for developers, but I can't remember where, and for what language.  

All of that said - I don't feel very strongly about it, I can switch this off in Hound if you prefer the visual style of `/* */`